src/prop-converter.js

```diff
@@ -41,16 +45,18 @@ function propConverter(ComposedComponent) {
       // child is passed, as well as if the child is falsy.
       workingProps.children = React.Children.toArray(props.children);
 
-      // FlipMove does not support stateless functional components.
-      // Check to see if any supplied components won't work.
-      // If the child doesn't have a key, it means we aren't animating it.
-      // It's allowed to be an SFC, since we ignore it.
-      const noStateless = workingProps.children.every(child =>
-         !isElementAnSFC(child) || typeof child.key === 'undefined'
-      );
-
-      if (!noStateless) {
-        console.warn(statelessFunctionalComponentSupplied());
+      if (process.env.NODE_ENV !== 'production') {
+        // FlipMove does not support stateless functional components.
+        // Check to see if any supplied components won't work.
+        // If the child doesn't have a key, it means we aren't animating it.
+        // It's allowed to be an SFC, since we ignore it.
+        const noStateless = workingProps.children.every(child =>
+           !isElementAnSFC(child) || typeof child.key === 'undefined'
+        );
+
+        if (!noStateless) {
+          console.warn(statelessFunctionalComponentSupplied());
+        }
```

Comment:
Can we move this whole block into a method on the class? Something like:

```js
checkForStatelessFunctionalComponents() {
  if (process.env.NODE_ENV === 'production') {
    return;
  }

  // insert the chunk of original code here, with the comment, and the console.warn.
}
```

Then we'd invoke this method on like 48, so that the behaviour isn't changed (still runs at the same time)